### PR TITLE
man: Specify that only d-bus methods can be answered async

### PR DESCRIPTION
**Pull request by igo95862**
https://github.com/systemd/systemd/pull/18206#issuecomment-800491578

Closes https://github.com/systemd/systemd/pull/18206 https://github.com/systemd/systemd/issues/18193

### CODE REVIEW

**Comment by igo95862** [comment]
I use the `<constant>` element to mark the **method** word. Not sure if that's correct. Maybe there is an element tag that just makes the text bold.

**Comment by poettering** [comment]
I think `<parameter>` would be the right tag for that.

**Comment by igo95862** [comment]
What needs a rework? @bluca

**Comment by yuwata** [comment]
> What needs a rework? @bluca

Please see @poettering's comment.

File: man/sd_bus_add_object.xml [hunk 1]
```diff
@@ -294,11 +294,12 @@
     returns zero, an error reply is sent back to the caller indicating no matching object for the
     request was found.</para>
 
-    <para>Note that you can return a positive integer from a callback without
+    <para>Note that you can return a positive integer from a <constant>method</constant> callback without
```
Comment (bluca) [inline man/sd_bus_add_object.xml]: @igo95862 the request is to use `<parameter>` instead of `constant`

**Comment by igo95862** [comment]
Ok done. I somehow did not realize that.

**Comment by poettering** [comment]
thanks so much!

**Pull request merged**